06CodeQuality/sec01WhyCareAboutQuality.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Why Care About Code Quality?
+---
+
+Programming is not just about producing code that works or passes tests. If code is not also usable, it will not be used. If code is not also maintainable, it will not be maintained. There are many aspects to code quality, the importance of which depends on the intended use of the code. A very dry list of code quality measures can be found in Steve McConnell's *Code Complete*:
```

Comment:
How about adding something along the lines that if software is not maintained, it will eventually become unusable?